src/main/java/bio/terra/workspace/service/workspace/WorkspaceService.java

```diff
@@ -70,9 +69,8 @@ public WorkspaceDescription getWorkspace(UUID id, AuthenticatedUserRequest userR
     return workspaceDao.getWorkspace(id);
   }
 
-  public void deleteWorkspace(UUID id, String userToken) {
+  public void deleteWorkspace(UUID id, AuthenticatedUserRequest userReq) {
```

Comment:
for consistency, I changed `deleteWorkspace` to accept a `AuthenticatedUserRequest`, since that's what the other two methods in this class accept.